packages/graph/hash_graph/bench/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "hash_graph_benches"
+version = "0.0.0"
+edition = "2021"
+publish = false
+autobenches = false
+
+[dev-dependencies]
+criterion = { version = "0.4.0", features = ["async_tokio"] }
+criterion-macro = "0.4.0"
+futures = "0.3.21"
+graph = { path = "../lib/graph", features = ["clap"] }
+graph-test-data = { path = "../tests/test_data" }
+rand = "0.8.5"
+serde = { version = "1.0.137", features = ["derive"] }
+serde_json = "1.0.83"
+tokio = { version = "1.18.2", features = ["rt-multi-thread", "macros"] }
+tokio-postgres = { version = "0.7.6", default-features = false }
+type-system = { git = "https://github.com/blockprotocol/blockprotocol", rev = "6526c0e" }
+uuid = { version = "1.1.2", features = ["v4", "serde"] }
+
+[[bench]]
+# TODO - Rename, lib.rs, main.rs, hash_graph_benchmark.rs?
+name = "entry"
```

Comment:
I couldn't come up with a good name for this. As expressed in the comment above `autobenches`, we have a single entrypoint to the benchmark 'binary' (even though we may have many benchmarks). 

This kind of makes me think `main.rs` won't be too misleading?

Reply:
One of my main aversions is I don't like `bench/benches/bench.rs` 😂 Not fully against it though